sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -152,7 +152,10 @@ class DateTime final : public _detail::Clock::time_point {
     AllDigits,
 
     /// Drop all the fractional time digits.
-    Truncate
+    Truncate,
+
+    /// Include only the date portion, skipping time and time zone.
+    Date,
```

Comment:
I think this should be `DateOnly`, which correctly reflects the behavior.

Note that if you specify `TimeFractionFormat::Date`, you are no longer generating valid Rfc3339 dates, but the documentation doesn't reflect that specifying `TimeFractionFormat::Date` breaks the `DateFormat` contract.

 As such, you should consider instead of using `TimeFractionFormat` as the mechanism to represent these, you might want to use `DateFormat::Rfc3339FullDate`.